.github/workflows/simorgh-misc-checks.yml

```diff
@@ -39,3 +39,6 @@ jobs:
 
       - name: Linting
         run: npm run test:lint
+
+      - name: NPM Audit
```

Comment:
Can we also add Audit to the name at the top?

Reply:
done